### PR TITLE
netware: remove leftover traces

### PR DESCRIPTION
**Pull request by danielgustafsson**
Commit 3b16575ae938dec2a29454631a12aa52b6ab9c67 removed support for building on Novell Netware, but a few leftover traces remained. This removes the last bits.

### CODE REVIEW

File: lib/nonblock.c [hunk 1]
```diff
@@ -31,9 +31,6 @@
 #include <fcntl.h>
 #endif
 
-#if (defined(HAVE_IOCTL_FIONBIO) && defined(NETWARE))
```
Comment (ryandesign) [inline lib/nonblock.c:34]: Could you keep this `#include <sys/filio.h>` but change the condition to `#ifdef HAVE_SYS_FILIO_H`? I'm trying to revive support for classic Mac OS and it needs this.

Comment (danielgustafsson) [reply lib/nonblock.c:34]: Couldn't your patch just as well include it if it needs it? I understand that it'll be a merge conflict once this goes in (if it does) but it should be trivially fixed.

**Review by bagder — APPROVED** [review]
:rocket:

File: lib/nonblock.c (hunk 1 above)
Comment (ryandesign) [reply lib/nonblock.c:34]: OK.

**Comment by danielgustafsson** [comment]
> OK.
> 
Also, as a sidenote, I think it’s great to see a stab at reviving MacOS support! Keep up the good work!

**Pull request closed without merging**